packages/abi-gen-wrappers/src/generated-wrappers/erc20_token.ts

```diff
@@ -583,6 +584,7 @@ export class ERC20TokenContract extends BaseContract {
         artifact: ContractArtifact | SimpleContractArtifact,
         supportedProvider: SupportedProvider,
         txDefaults: Partial<TxData>,
+        artifactDependencies: { [contractName: string]: ContractArtifact | SimpleContractArtifact },
```

Comment:
Shouldn't this be named `contractDependenciesArtifacts`? You are passing in the artifacts of the dependencies of the contract no? `artifactDependencies` made me think these are dependencies of the artifacts.